app/src/main/java/com/concordia/mcga/helperClasses/OutdoorDirections.java

```diff
@@ -0,0 +1,168 @@
+package com.concordia.mcga.helperClasses;
+
+import android.content.Context;
+
+import com.akexorcist.googledirection.constant.TransportMode;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.model.LatLng;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by taimoorrana on 2017-03-25.
+ */
+
+public class OutdoorDirections {
+    private static OutdoorDirections outdoorDirections;
+    private List<OutdoorDirection> outdoorDirectionList;
+    private String selectedTransportMode = null;
+    private OutdoorDirection selectedOutdoorDirection;
+
+    private OutdoorDirections() {
+        outdoorDirectionList = new ArrayList<>();
+        List<String> transportModes = new ArrayList<String>() {{
+            add(TransportMode.BICYCLING);
+            add(TransportMode.DRIVING);
+            add(TransportMode.TRANSIT);
+            add(TransportMode.WALKING);
+        }};
+        int TOTAL_OUTDOOR_DIRECTION = 4;
```

Comment:
This constructor is really ugly. And it betrays the intent of this class which is just to store `OutdoorDirection` objects...